app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -108,4 +108,19 @@ def filter_allowed_hosts(all_hosts)
     ovirt_services = ManageIQ::Providers::Redhat::InfraManager::OvirtServices::Builder.new(ems).build(:use_highest_supported_version => true).new(:ems => ems)
     ovirt_services.filter_allowed_hosts(self, all_hosts)
   end
+
+  def validate_memory_limit(_field, values, dlg, fld, _value)
```

Comment:
usually, generic validation methods are defined in a higher parent-class, however, since this method contains a specific RHV logic, I defined it in this file.